packages/aws-cdk/test/api/hotswap/lambda-hotswap-deployments.test.ts

```diff
@@ -349,23 +444,6 @@ test("will not perform a hotswap deployment if it doesn't know how to handle a s
 
   // THEN
   await expect(() =>
-    tryHotswapDeployment(mockSdkProvider, {}, currentCfnStack, cdkStackArtifact),
+    setup.tryHotswapDeployment({}, cdkStackArtifact),
   ).rejects.toThrow("We don't support the 'UnknownAttribute' attribute of the 'AWS::S3::Bucket' resource. This is a CDK limitation. Please report it at https://github.com/aws/aws-cdk/issues/new/choose");
 });
```

Comment:
General comment in this file: let's try to keep the tests in the same order as they were in the file before, and then add the new tests at the end. This way, the diff will be much easier to review.